CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: 3 more joke hints.
 - Added: The Hints Spoiler tab now indicates which location a hint is pointing to.
 - Added: Experimental option under Generation -> Logic Settings that improves the results of generation but increases generation time.
+- Added: Items now have an extra category to discern if it is a valid starting item.
```

Comment:
the actual user-facing change here is that those echoes items can't be starting any more. the changelog entry should be under echoes and say something like `- Fixed: Cannon Ball and the multiplayer pickups can no longer be selected as starting items, since they did not work in-game when doing so.`